force-app/main/default/classes/HH_Container_LCTRL.cls

```diff
@@ -514,12 +535,16 @@ public with sharing class HH_Container_LCTRL {
 
     @AuraEnabled
     public static Map<String, String> getFieldLabels() {
+        String undeliverableAddressLabel = canReadContactUndeliverable ?
+            UTIL_Describe.getFieldLabel(String.valueOf(Contact.SObjectType), String.valueOf(Contact.Undeliverable_Address__c)) : '';
+        String undeliverableLabel = canReadAddressUndeliverable ?
+            UTIL_Describe.getFieldLabel(String.valueOf(Address__c.SObjectType), String.valueOf(Address__c.Undeliverable__c)) : '';
+
         Map<String, String> fieldLabelsBySObject = new Map<String, String> {
-            String.valueOf(Contact.Undeliverable_Address__c) => UTIL_Describe.getFieldLabel(
-                    String.valueOf(Contact.SObjectType), String.valueOf(Contact.Undeliverable_Address__c)),
-            String.valueOf(Address__c.Undeliverable__c) => UTIL_Describe.getFieldLabel(
-                    String.valueOf(Address__c.SObjectType), String.valueOf(Address__c.Undeliverable__c))
+            'Undeliverable_Address__c' => undeliverableAddressLabel,
```

Comment:
I changed this to avoid additional permissions checking in the front end, the Label will just be blank in cases where permissions are missing.